frontend/src/index/css/App.css

```diff
@@ -16,3 +16,12 @@
 .App-link {
     color: #61dafb;
 }
+
+main {
+    max-width: 75%;
+    margin-left: auto;
+    margin-right: auto;
```

Comment:
FYI: margin: 0 auto - means 0 on top and bottom, auto left and right. In case you want to write it shorter

Reply:
Great tip